server/src/main/java/org/elasticsearch/action/search/ClearScrollResponse.java

```diff
@@ -30,8 +30,8 @@
     private static final ParseField SUCCEEDED = new ParseField("succeeded");
     private static final ParseField NUMFREED = new ParseField("num_freed");
 
-    private static final ConstructingObjectParser<ClearScrollResponse, Void> PARSER = new ConstructingObjectParser<>("clear_scroll",
-            true, a -> new ClearScrollResponse((boolean)a[0], (int)a[1]));
+    private static final ConstructingObjectParser<ClosePointInTimeResponse, Void> PARSER = new ConstructingObjectParser<>("clear_scroll",
```

Comment:
Can you please explain what's the need for this change?

Reply:
This is a bit messy. ClosePointInTimeResponse extends ClearScrollResponse, and this change just ensures that `fromXContent` method returns an instance of ClosePointInTimeResponse. I will work on a small PR to clean up this.